wasm.md

```diff
@@ -287,6 +296,22 @@ The target type is before the `.`, and the source type is after the `_`.
 Numeric Operators
 -----------------
 
+These instructions are reduced to from the plain instructions defined above.
+They take operands as parameters rather than from the `<valstack>` and is used to define the numeric computation rules.
+
+```k
+    syntax Instr  ::= IValType "." IUnOp   Int
+                    | IValType "." IBinOp  Int   Int
+                    | IValType "." ITestOp Int
+                    | IValType "." IRelOp  Int   Int
+                    | IValType "." ConvOp  Int
+ //                 | FValType "." FUnOp   Float
+ //                 | FValType "." FBinOp  Float Float
+ //                 | FValType "." FTestOp Float
+ //                 | FValType "." FRelOp  Float Float
+ //                 | FValType "." ConvOp  Float
+```
```

Comment:
Let's move these back to where their other productions are defined above. Make sure the formatting is nice.

```
syntax PlainInstr ::= IValType "." IUnOp
syntax Instr ::= IValType "." IUnOp Int
```

Reply:
fixed